pkg/virt-controller/watch/node_test.go

```diff
@@ -14,21 +14,21 @@ import (
 
 	appv1 "k8s.io/api/apps/v1"
 	k8sv1 "k8s.io/api/core/v1"
-	v1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
```

Comment:
78426b46ec474ef852df04d9287cba79135c621e please drop the link as it can move. The "Following coding conventions" should be enough.

Reply:
:+1: Done